nservicebus/azure-service-bus/dlq-forwarding.md

```diff
@@ -0,0 +1,28 @@
+---
+title: Azure Service Bus DLQ Forwarding
+versions: '[7,)'
+reviewed: 2016-08-29
+tags:
+- Azure
+- Cloud
+- Error Handling
+---
+
+## Dead Letter Queue Forwarding
+
+When Azure Service Bus service detects that an endpoint cannot process messages, or when it has been configured not to discard expired or filtered messages, then it will move those messages to a dead letter queue associated with the entity, originally contained those messages.
```

Comment:
>a dead letter queue associated with the entity, originally contained those messages.

isn't very clear. 

How about:

When Azure Service Bus service detects that an endpoint cannot process messages, or when it has been configured not to discard expired or filtered messages, then it will move those messages to the configured dead letter queue associated with the endpoint?